Bugzilla/Config/Auth.pm

```diff
@@ -107,18 +107,41 @@ sub get_param_list {
         },
 
         {
-            name    => 'password_complexity',
-            type    => 's',
-            choices => [ 'no_constraints', 'bmo' ],
-            default => 'no_constraints',
-            checker => \&check_multi
-        },
-
-        {
             name    => 'password_check_on_login',
             type    => 'b',
             default => '1'
         },
+
+        {
+            name => 'passwdqc_min',
+            type => 't',
+            default => 'undef, 24, 11, 8, 7',
```

Comment:
this needs a checker to ensure the correct number of elements are provided, and that each element is either "undef" or an int.